TransbankPosSDK/Responses/SaleResponse.cs

```diff
@@ -0,0 +1,116 @@
+            Last4Digits = fields[ParameterMap["Last4Digits"]].Trim() != "" ? int.Parse(fields[ParameterMap["Last4Digits"]]) : 0;
+            OperationNumber = fields[ParameterMap["OperationNumber"]].Trim() != "" ? int.Parse(fields[ParameterMap["OperationNumber"]]) : 0;
+            CardType = fields[ParameterMap["CardType"]].Trim();
+
+            string date = fields[ParameterMap["AccountingDate"]].Trim();
+            AccountingDate = null;
+            if (date != "")
+            {
+                AccountingDate = DateTime.ParseExact(date, "ddMMyyyy", DateTimeFormatInfo.InvariantInfo);
+            }
+
+            AccountNumber = fields[ParameterMap["AccountNumber"]].Trim() == "" ? 0 : long.Parse(fields[ParameterMap["AccountNumber"]]);
+            CardBrand = fields[ParameterMap["CardBrand"]].Trim();
+
+            date = fields[ParameterMap["RealDate"]].Trim();
+            string hour = fields[ParameterMap["RealTime"]].Trim();
+
+            RealDate = null;
+            if (date + hour != "")
+            {
+                RealDate = DateTime.ParseExact(date + hour, "ddMMyyyyHHmmss", DateTimeFormatInfo.InvariantInfo);
+            }
+              
+            EmployeeId = fields[ParameterMap["EmployeeId"]].Trim() == "" ? int.Parse(fields[ParameterMap["EmployeeId"]]) : 0;
+            Tip = fields[ParameterMap["Tip"]].Trim() != "" ? int.Parse(fields[ParameterMap["Tip"]]) : 0;
+        }
+
+        public override string ToString()
+        {
+            string accountdate = AccountingDate.HasValue ? AccountingDate.Value.ToString("dd/MM/yyyy hh:mm:ss") : "";
+            string realdate = RealDate.HasValue ? RealDate.Value.ToString("dd/MM/yyyy hh:mm:ss") : "";
+            return "Function: " + FunctionCode + "\n" +
+                   "Response: "+ ResponseMessage + "\n" +
+                   "Commerce Code: " + CommerceCode + "\n" +
+                   "Terminal Id: " + TerminalId + "\n" +
+                   "Ticket: " + Ticket + "\n" +
+                   "Autorization Code: " + AutorizationCode + "\n" +
+                   "Ammount: " + Ammount + "\n" +
+                   "Shares Number: " + SharesNumber + "\n" +
+                   "Shares Amount: " + SharesAmount + "\n" +
+                   "Last 4 Digits: " + Last4Digits + "\n" +
+                   "Operation Number: " + OperationNumber + "\n" +
+                   "Card Type: " + CardType + "\n" +
+                   "Accounting Date: " + accountdate + "\n" +
+                   "Account Number: " + AccountNumber + "\n" +
+                   "Card Brand: " + CardBrand + "\n" +
+                   "Real Date: " + realdate + "\n" +
+                   "Employee Id: " + EmployeeId + "\n" +
+                   "Tip: " + Tip;
+        }
+
+        private static LoadKeyCloseResponse CreateBase(string cresponse)
+        {
+            return new LoadKeyCloseResponse
+            {
+                function = int.Parse(cresponse.Substring(1, 4)),
```

Comment:
better lets call *functionCode* ? (*function* is a private name)

Reply:
function is not a reserved word in c#, but i think it is still better to rename to Command. This change has to be done in the C Library since this name came from the LoadKeyCloseResponse Struct that has to be renamed to BaseResponse Struct.